tools/generateIndices.js

```diff
@@ -0,0 +1,82 @@
+const path = require('path');
+const fs = require('fs');
+const _ = require('lodash');
+const posix = path.posix;
+
+const contents = [`
+/**
+ * THIS IS A GENERATED FILE
+ *
+ * Please see /tools/generateIndices.js to regenerate after structural changes or component additions / deletions
+ */
+
+`];
+const exportItems = [];
+
+const src = path.resolve(__dirname, '..', 'src');
+
+const appendExports = (dir, subDirs, files) => {
```

Comment:
FYI [Future]: This should likely be converted to a webpack plugin since this is the sort of thing it handles, then you would receive a stream of files that you could generate the indices off of. It would allow this solution to be a little more flexible.